src/NuGet/Microsoft.CodeAnalysis.Common.nuspec

```diff
@@ -16,7 +16,7 @@
     <dependencies>
       <group targetFramework="netstandard1.3">
         <dependency id="Microsoft.CodeAnalysis.Analyzers"              version="$MicrosoftCodeAnalysisAnalyzersVersion$" />
-
+        <dependency id="Microsoft.DiaSymReader.Native"                 version="$MicrosoftDiaSymReaderNativeVersion$" />
```

Comment:
Paging @tmat to confirm this is right.

Reply:
I don't think this should be here.

Reply:
The compiler doesn't have a hard dependency on DSRN. The host/app should add that dependency if it needs to emit Windows PDB.

Reply:
The CodeAnalysis.csproj has 
<PackageReference Include="Microsoft.DiaSymReader.Native" Version="$(MicrosoftDiaSymReaderNativeVersion)" />  

And I saw the code imports this dll, does that mean we don't need to list this kind of dependency in .nuspec? Thanks

Reply:
@tmat The CodeAnalysis.csproj has
PackageReference Include="Microsoft.DiaSymReader.Native" Version="$(MicrosoftDiaSymReaderNativeVersion)"
And I saw the code imports this dll, does that mean we don't need to list this kind of dependency in .nuspec? Thanks

Reply:
The compiler doesn't _require_ it, it's an optional dependency that's picked up if available. Shouldn't be in nuspec of this package. It is included in `Microsoft.Net.Compilers`.

Reply:
thanks, I removed it now